src/main/java/edu/ucsb/cs156/dining/controllers/ReviewController.java

```diff
@@ -117,6 +117,13 @@ public Review postReview(
         MenuItem reviewedItem = menuItemRepository.findById(itemId).orElseThrow(
                 () -> new EntityNotFoundException(MenuItem.class, itemId)
         );
+
+        // if (review.getReviewerComments() == null) {
+        //     // review.setStatus(ModerationStatus.APPROVED);
+        // } else {
+        //     // review.setStatus(ModerationStatus.AWAITING_REVIEW);
+        // }
+
```

Comment:
Commented out code does not belong in a PR targetting main, and this is not related to this PR.